common/src/main/java/org/jboss/hal/testsuite/cli/ServerManager.java

```diff
@@ -82,5 +82,19 @@ public String getVersion() {
      * @return true if server becomes available before timeout passes, false otherwise
      */
     public abstract boolean waitUntilAvailable(long timeout);
+
+    /**
+     * Reloads server(s) and waits until server(s) is(are) running again
+     *
+     * @param timeout max time to wait for server to start, in milliseconds
+     */
+    public abstract void reloadAndWaitUntilRunning(long timeout);
+
+    /**
+     * Reloads server(s), which require reload and waits until server(s) is(are) running again
+     *
+     * @param timeout max time to wait for server to start, in milliseconds
+     */
+    public abstract void reloadIfRequiredAndWaitUntilRunning(long timeout);
```

Comment:
Don't see any reason for this method as CliClient does this by default, see `org.jboss.hal.testsuite.cli.CliClient#reload()`